src/VirtoCommerce.Platform.Data/Infrastructure/DbContextUnitOfWork.cs

```diff
@@ -27,21 +27,12 @@ public DbContextUnitOfWork(DbContext context)
 
         public int Commit()
         {
-            var result = DbContext.SaveChanges();
-            return result;
+            return DbContext.SaveChanges();
         }
 
-        public async Task<int> CommitAsync()
+        public Task<int> CommitAsync()
         {
-            try
-            {
-                var result = await DbContext.SaveChangesAsync();
-                return result;
-            }
-            catch (Exception ex)
-            {
-                throw new PlatformException(ex.ExpandExceptionMessage());
-            }
+            return DbContext.SaveChangesAsync();
```

Comment:
Why `await` is removed here?
Now even if CommitAsync will be awaited, it will not wait for SaveChangesAsync completion, isn't it?

Reply:
My wrong, SaveChangesAsync will be awaited in that case.